cli/flags.rs

```diff
@@ -23,6 +29,7 @@ pub enum DenoSubcommand {
   Compile {
     source_file: String,
     output: Option<PathBuf>,
+    args: Vec<String>,
```

Comment:
This is surprising; it means compiled binaries can't have runtime args. What's the use case for having pre compiled CLI args?

Reply:
> This is surprising; it means compiled binaries can't have runtime args. What's the use case for having pre compiled CLI args?

No, the runtime args are _appended_ to the ones passed at compile-time, like with `deno install`.

Reply:
Okay, good to know, can you update docs page as well?